src/components/ProductList/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function ProductList({ values }) {
```

Comment:
I like the component name, nice work!

Let's rename this prop to something more meaningful instead as `values` is just too generic. In the end, _anything_ is a value 😅. Maybe as this component is named `ProductList` that prop could be renamed to `products`?

Reply:
Yes, you're right! It's easier to understand `product` versus `values`. What's values? 😄  ... thanks!